bsp/boards/scum/board_info.h

```diff
@@ -33,8 +33,8 @@ to return the board's description.
 // ==== SCuM RF timer specific 
 
 /** 
-* since SCuM uses 500KHz, but the statck protocol is designed for 32KHz.
-* the following two marco are used to convert the couter value bewteen
+* since SCuM uses 500KHz, but the stack protocol is designed for 32KHz.

```

Comment:
sorry to be a pain. KHz are "Kelvin Hertz". You probably mean kHz or "kilo-Hertz"